No package files given... Grabbing a set.
--- !found   obj /usr/share/man/man8/tdbtool.8.bz2
--- !found   obj /usr/share/man/man8/tdbrestore.8.bz2
--- !found   obj /usr/share/man/man8/tdbdump.8.bz2
--- !found   obj /usr/share/man/man8/tdbbackup.8.bz2
--- !found   obj /usr/lib64/pkgconfig/tdb.pc
--- !found   obj /usr/lib64/libtdb.so.1.4.3
--- !found   sym /usr/lib64/libtdb.so.1
--- !found   sym /usr/lib64/libtdb.so
--- !found   obj /usr/lib/python3.8/site-packages/tdb.cpython-38-x86_64-linux-gnu.so
--- !found   obj /usr/lib/python3.8/site-packages/_tdb_text.py
--- !found   obj /usr/include/tdb.h
--- !found   obj /usr/bin/tdbtool
--- !found   obj /usr/bin/tdbrestore
--- !found   obj /usr/bin/tdbdump
--- !found   obj /usr/bin/tdbbackup
--- !empty   dir /usr/share/man/man8
--- !empty   dir /usr/share/man
--- !empty   dir /usr/share
--- !empty   dir /usr/lib64/pkgconfig
--- !empty   dir /usr/lib64
--- !empty   dir /usr/lib/python3.8/site-packages
--- !empty   dir /usr/lib/python3.8
--- !empty   dir /usr/lib
--- !empty   dir /usr/include
--- !empty   dir /usr/bin
--- !empty   dir /usr
>>> Regenerating /etc/ld.so.cache...
